TDD_Training/FindMultiplesTests/FindMultiplesTests.cs

```diff
@@ -0,0 +1,90 @@
+using System;
+using TDD_Training;
+using Xunit;
+
+namespace FindMultiplesTests
+{
+    public class FindMultiplesTests
+    {
+        private readonly FindMultiples findMultiplesService = new FindMultiples();
```

Comment:
readonly fields naming convention 
_findMultiplesService

Reply:
Got it. I will update in the code